### PR TITLE
Gem minor version updates incl faraday, aws-sdk and fog

### PR DESCRIPTION
**Pull request by dcorking**
## Status

* Current Status: Ready for tech review
* Review App link(s): https://teachcomputing-pr-1744.herokuapp.com/

## Review progress:

- [ ] Tech review completed

## What's changed?

- All updates available with `bundle update`
- Remove unsed VCR testing gem

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -547,18 +561,19 @@ GEM
     sitemap_generator (6.0.2)
       builder (~> 3.0)
     smart_properties (1.17.0)
-    solargraph (0.48.0)
+    solargraph (0.49.0)
```
Comment (Caleb-T-Owens) [inline Gemfile.lock:564]: Is there a reason why solargraph is a gem installed in this project. AFAIK this is a language server and should be installed globally on a developers system

File: Gemfile.lock [hunk 2]
```diff
@@ -591,9 +606,9 @@ GEM
       unf_ext
     unf_ext (0.0.8.2)
     unicode-display_width (2.4.2)
-    vcr (6.1.0)
-    view_component (2.74.1)
-      activesupport (>= 5.0.0, < 8.0)
+    vcr (6.2.0)
```
Comment (Caleb-T-Owens) [inline Gemfile.lock:609]: Looking at the VCR folder and seeing that there are no references to `VCR`, we have no tests which make use of this gem. Is there much point in having this gem until we really need it?

When it comes to testing using VCR always seems like a last resort to me when it comes to unit testing as it requires regular maintenance and does a poor job of facilitating just the "unit" which should really be the fiddling of data before its passed to the api and fiddling of the response.

**Comment by dcorking** [comment]
☝️ Github thinks that build_and_test failed, but click through to the latest CI run and you'll see that it passed: https://app.circleci.com/pipelines/github/NCCE/teachcomputing.org/9698/workflows/47a4cd8e-dda6-4e50-8467-10ce482679b0 (on commit https://github.com/NCCE/teachcomputing.org/commit/680870501e3fdf8f0bdc6e24c581fd61aacbe296 )

File: Gemfile.lock (hunk 1 above)
Comment (dcorking) [reply Gemfile.lock:564]: Yes. Reason:
>  If your project uses Bundler, the most comprehensive way to use your bundled gems is to bundle Solargraph.
https://marketplace.visualstudio.com/items?itemName=castwide.solargraph

**Review by Caleb-T-Owens — APPROVED** [review]

**Pull request merged**